### PR TITLE
Miner version collection

### PR DESCRIPTION
**Pull request by emnoor-reef**

### CODE REVIEW

File: miner/envs/runner/nginx/build-image.sh [hunk 1]
```diff
@@ -2,4 +2,4 @@
 set -eux -o pipefail
 
 IMAGE_NAME="backenddevelopersltd/compute-horde-miner-nginx:v0-latest"
-docker build -t $IMAGE_NAME .
+docker build --platform=linux/amd64 -t "$IMAGE_NAME" .
```
Comment (andreea-popescu-reef) [inline miner/envs/runner/nginx/build-image.sh:5]: why is this added?

**Review by andreea-popescu-reef — APPROVED** [review]

File: miner/envs/runner/nginx/build-image.sh (hunk 1 above)
Comment (emnoor-reef) [reply miner/envs/runner/nginx/build-image.sh:5]: Well my machine is arm and I accidentally pushed an arm image before. So I add this to prevent myself from ruining other people's day again. :P
